### DISCUSSION TITLE
#58 togETHer biweekly meeting #43

### DISCUSSION THREAD

**Discussion by bambalados** [tog-ETH-er/togETHer#43] [General]
Agenda:

- Community Updates
- togETHer X ETH Shanghai Web3 Summit
https://www.notion.so/1b0bafdbeb594309ac2aea2ce6f64a76 ｜ https://www.shanghai-upgrade.org/
(Is togETHer willing to become a supporting community? Vote?)
- Fireside Chat

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/uzr-vzib-znh

**Comment by bambalados** [comment]
**Crypto Tech Night**

- CTN#20: Web3在中国的发展和机遇。b站视屏: [link](https://www.bilibili.com/video/BV11L411Z7Hw/?share_source=copy_web&vd_source=63b525014af4d11ec60800f5d2ca8c55)
- 规划CTN#21 - 环绕Shanghai Upgrade的活动

**Comment by Stamp9** [comment]
## 原语里弄

进行课题研究。

**Comment by liushooter** [comment]
## rebase

- web3 极客日报 最近 14 期 http://daily.rebase.network/

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 日常翻译进行中

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

3个PR #931 ～ #933

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《MEV101》by Zeo
  - https://www.youtube.com/watch?v=QcfWrRmhXls
 
- 《去信任的跨链演进-从 MPC 到 ZeroKnowledge》by Frank
  - https://www.youtube.com/watch?v=WFLPfPhU18M
  - https://www.youtube.com/watch?v=BbKsJFxLfeg
